apps/glusterfs/app_node.go

```diff
@@ -226,9 +226,10 @@ func (a *App) NodeDelete(w http.ResponseWriter, r *http.Request) {
 		}
 
 		// Check the node can be deleted
-		if !node.IsDeleteOk() {
-			http.Error(w, ErrConflict.Error(), http.StatusConflict)
```

Comment:
@NehaRawat I have been looking at this a little more, and have determined that the error returned by the server should be fixed also.

When I wrote this, I didn't know about the call `fmt.Errorf(...)` (even though it is not useful exactly here.  Here instead we will use fmt.Sprintf() since we need to return a string instead of an error). Now I am starting to use that instead of `ErrXXX` errors.

I suggest something like the following:

```go
if !node.IsDeleteOk() {
    http.Error(w, fmt.Sprintf("Unable to delete node [%v] because it contains volumes", id), http.StatusConflict)
}
```